qos_bandwidth.adoc

```diff
@@ -149,8 +149,8 @@ listed in <<BC_MON_OP>>.
 | --           | 24-31     | Designated for custom use.
 |===
 
-The `EVT_ID` field is used to program, using the `CONFIG_EVENT` operation, the
-identifier of the event to count in the monitoring counter selected by `MCID`.
+The `EVT_ID` field uses the `CONFIG_EVENT` operation to program the
```

Comment:
@kersten1 The `CONFIG_EVENT` operation uses the `EVT_ID` operand to program the identifier of the event to count in the monitoring counter. The change seems to change the semantics.